### ISSUE TITLE
[MAS1.4.1] Links present in 'Emulator Settings' pane are identifiable as link by their color only.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#739]
**Actual Result:**
The only way to know the difference between links and normal text present in 'Emulator Settings' pane is Blue color of those links.

**Expected Result:**
There should be some other way as well to know the difference between links and normal text present in 'Emulator Settings' pane beside Blue color of those links. All links must be underlined.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome screen will open up.
3. Navigate in Left side pane, reach at Settings button and press Enter.
4. Notice 'ngrok' and 'wiki page' links (in blue) present on this page.

**User Impact**:
Users with low vision or color-blindness will find it difficult to identify these links.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 1.4.1 – Use of Color [(16)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579})

**Attachments:**
![mas1 4 1_botframeworkemulator_emulatorsettings_links](https://user-images.githubusercontent.com/41668935/43773960-133c2056-9a65-11e8-90cb-f9a12596b8dc.png)
